openbb_terminal/mutual_funds/mutual_fund_controller.py

```diff
@@ -137,23 +106,423 @@ def print_help(self):
         mt.add_raw("\n")
         mt.add_param("_fund", fund_string)
         mt.add_raw("\n")
-        mt.add_cmd("info", self.fund_symbol)
         mt.add_cmd("plot", self.fund_symbol)
-        if self.country == "united states":
-            mt.add_cmd("sector", self.fund_symbol)
-            mt.add_cmd("equity", self.fund_symbol)
-        if self.country == "sweden":
-            mt.add_cmd("alswe", self.fund_symbol)
-            mt.add_cmd("infoswe", self.fund_symbol)
-            mt.add_cmd("forecast", self.fund_symbol)
+        mt.add_cmd("sector", self.fund_symbol)
+        mt.add_cmd("holdings", self.fund_symbol)
+        mt.add_cmd("carbon_metrics", self.fund_symbol)
+        mt.add_cmd("exclusion_policy", self.fund_symbol)
+
+        # if self.country == "sweden":
+        #     mt.add_cmd("alswe", self.fund_symbol)
+        #     mt.add_cmd("infoswe", self.fund_symbol)
+        #     mt.add_cmd("forecast", self.fund_symbol)
         console.print(text=mt.menu_text, menu="Mutual Funds")
 
     def custom_reset(self):
         """Class specific component of reset command"""
-        if self.fund_name:
-            return ["funds", f"load {self.fund_name} --name"]
+        if self.fund_symbol:
+            return ["funds", f"load {self.fund_symbol}"]
+            # return ["funds", f"load {self.fund_name} --name"]
         return []
 
+    # @log_start_end(log=logger)
+    # def call_country(self, other_args: List[str]):
+    #     """Process country command"""
+    #     parser = argparse.ArgumentParser(
+    #         add_help=False,
+    #         formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    #         prog="country",
+    #         description="Set a country for funds",
+    #     )
+    #     parser.add_argument(
+    #         "-n",
+    #         "--name",
+    #         type=str,
+    #         dest="name",
+    #         nargs="+",
+    #         help="country to select",
+    #     )
+    #     if other_args and "-" not in other_args[0][0]:
+    #         other_args.insert(0, "-n")
+    #     ns_parser = self.parse_known_args_and_warn(parser, other_args)
+    #     if ns_parser:
+    #         country_candidate = " ".join(ns_parser.name)
+    #         if country_candidate.lower() in self.fund_countries:
+    #             self.country = " ".join(ns_parser.name)
+    #         else:
+    #             console.print(
+    #                 f"{country_candidate.lower()} not a valid country to select."
```

Comment:
We would prefer to just get rid of everything and not comment it out.  We can always get it back from the git history